### PR TITLE
Adds "Redesign Your Logo" by Lemon Demon to the lobby music

### PR DESCRIPTION
**Pull request by MrHorizons**
[Lemon Demon - Redesign Your Logo](https://youtu.be/qImHuiYnVQ0)

Aside from "sounding cool" (as many lobby music additions are based on), I think the synth-heavy sounds and corporate surrealism jive very well with the tone and style of SS13, especially Yog. (see: that one round where Rotciv got brainwashed by Donk Co.)

#### Changelog

:cl:  
rscadd: Adds "Redesign Your Logo" by Lemon Demon to the lobby music.
/:cl:

### CODE REVIEW

**Review by jcatjmeow — APPROVED** [review]
Think about it this way
Guided evolution
This will be a new breed
Cutting edge but classic
Fully complimentary
Bold and self-sufficient
Symbolizing freedom
And, of course, your product
When the logo's finished
Then we will unveil it
Seven billion people
Instantly enlightened
Everyone's rebranded
All the fighting's over
Say goodbye to conflict
That design's rejected
There's no longer language
Only recognition
There's no longer country
Only corporate colors
Color makes us hungry
Hunger makes us human
Everyone can see it
Everything's connected

**Comment by ynot01** [comment]
are they even human?

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -33,7 +33,8 @@
 		"https://www.youtube.com/watch?v=RnQofA9CNww",						// minomus of DOMU - Winds of Fjords
 		"https://www.youtube.com/watch?v=xhlH91k-86E",						// J.G. Thirlwell - In a Spaceage Mood
 		"https://www.youtube.com/watch?v=Ld6TfpgJg7g",						// Tom Kane - Freeway Jazz
-		"https://www.youtube.com/watch?v=ZhhQrFfzFM4")						// Carpenter Brut - Escape from Midwich Valley
+		"https://www.youtube.com/watch?v=ZhhQrFfzFM4",						// Carpenter Brut - Escape from Midwich Valley
+		"https://www.youtube.com/watch?v=qImHuiYnVQ0")						// Lemon Demon - Redesign Your Logo
```
Comment (jcatjmeow) [inline yogstation/code/controllers/subsystem/ticker.dm]:
```suggestion
		"https://www.youtube.com/watch?v=HXk-0_rETJ8")						// Lemon Demon - Redesign Your Logo
```
This version is higher quality

**Review by Partheo — CHANGES_REQUESTED** [review]
Decent lobby theme, the repetitive synth beat feels like a cheese grater on my ears though. Waiting to get more opinions.

**Review by jcatjmeow — APPROVED** [review]

**Comment by jcatjmeow** [comment]
restore your branch, coward

**Pull request closed without merging**